src/packages/client/scripts/postinstall.js

```diff
@@ -322,8 +325,6 @@ function parsePackageManagerName(userAgent) {
   return packageManager
 }
 
-// prettier-ignore
-const UNABLE_TO_FIND_POSTINSTALL_TRIGGER__EMPTY_STRING = 'UNABLE_TO_FIND_POSTINSTALL_TRIGGER__EMPTY_STRING'
```

Comment:
Is this not used?

Reply:
I could not find it used anywhere, I guess it was used before and not anymore.

Reply:
Ping @jasonkuhrt - leftover?

Reply:
Hey, probably not a big deal